tools/packaging/kata-deploy/local-build/kata-deploy-binaries.sh

```diff
@@ -237,14 +240,20 @@ install_cc_image() {
 	local component="rootfs-image"
 	local root_hash_vanilla="root_hash_vanilla.txt"
 	local root_hash_tdx=""
+	local kernel_last_commit=""
 	if [ -n "${tee}" ]; then
+		jenkins="${jenkins_url}/job/kata-containers-2.0-rootfs-image-${tee}-cc-$(uname -m)/${cached_artifacts_path}"
 		if [ "${tee}" == "tdx" ]; then
-			jenkins="${jenkins_url}/job/kata-containers-2.0-rootfs-image-${tee}-cc-$(uname -m)/${cached_artifacts_path}"
 			component="${tee}-rootfs-image"
 			root_hash_vanilla=""
 			root_hash_tdx="root_hash_${tee}.txt"
 		fi
-	fi
+		if [ "${tee}" == "sev" ]; then
+			component="${tee}-rootfs-initrd"
+			root_hash_vanilla=""
+			kernel_last_commit="-$(get_kernel_image_name)"
```

Comment:
This is not the last commit in the parts related to the kernel, as in, patches added, or config bumped.

This is rather the last commit in the scripts used to build the kernel, so I am not sure how valid this part is.